pkg/config/imports.go

```diff
@@ -0,0 +1,402 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	clog "github.com/charmbracelet/log"
```

Comment:
What's this `clog` about? This should be getting lunged as an error, only permitting the log `log` alias.